lib/promscrape/discovery/consul/consul.go

```diff
@@ -10,18 +10,18 @@ import (
 //
 // See https://prometheus.io/docs/prometheus/latest/configuration/configuration/#consul_sd_config
 type SDConfig struct {
-	Server       string              `yaml:"server"`
+	Server       string              `yaml:"server,omitempty"`
 	Token        *string             `yaml:"token"`
 	Datacenter   string              `yaml:"datacenter"`
-	Scheme       string              `yaml:"scheme"`
+	Scheme       string              `yaml:"scheme,omitempty"`
 	Username     string              `yaml:"username"`
 	Password     string              `yaml:"password"`
 	TLSConfig    *promauth.TLSConfig `yaml:"tls_config"`
-	Services     []string            `yaml:"services"`
-	Tags         []string            `yaml:"tags"`
+	Services     []string            `yaml:"services,omitempty"`
+	Tags         []string            `yaml:"tags,omitempty"`
 	NodeMeta     map[string]string   `yaml:"node_meta"`
```

Comment:
It looks like `node_meta` is optional here

Reply:
fixed